httperfpy/httperfpy.py

```diff
@@ -117,55 +116,146 @@ def __params(self):
             "wset": None
         }
 
-
 class HttperfParser(object):
 
-    @classmethod
-    def parse(self, result_string):
+    def parse(self, result_string, options = {}):
```

Comment:
Could you add some doc/comment about what `options` could be -- I know that right now it just allows seeing the transitions but even that might be useful for others when debugging.

Reply:
will do.

Reply:
done.